custom/emails/magic-link/html.ejs

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <style>
+    body {
+      margin: 0;
+      font-family: sans-serif;
+    }
+
+    button {
+      color: white;
+      margin: 0.5em 1em;
+      border-radius: 3px;
+      padding: 0.25em 1em;
+      background: palevioletred;
+      border: 2px solid palevioletred;
+    }
+  </style>
+</head>
+
+<body>
+  <h2>Hello <%= display_name %>
+  </h2>
+  <p>Please confirm your magic link <%= action %> by clicking the below button:</p>
+  <a href="<%= url %>/auth/magic-link?token=<%= token %>&action=<%= action.replace(/ /g, '-') %>">
+    <button>Confirm <%= action %></button></a>
+  <p>Thanks,<br />The Team</p>
+</body>
```

Comment:
You can also copy & paste this URL into your browser:

<%= url %>/auth/magic-link?token=<%= token %>&action=<%= action.replace(/ /g, '-') %>